server/player/handler.go

```diff
@@ -68,6 +68,8 @@ type Handler interface {
 	// HandleBlockPick handles the player picking a specific block at a position in its world. ctx.Cancel()
 	// may be called to cancel the block being picked.
 	HandleBlockPick(ctx *event.Context, pos cube.Pos, b world.Block)
+	// HandleTrampleCrop handles the player trampling a crop.
+	HandleTrampleCrop(ctx *event.Context)
```

Comment:
Would maybe rename this to HandleCropTrample to be more in line with the other method names.